scuba/config.py

```diff
@@ -129,25 +129,58 @@ def _process_script_node(node, name):
     raise ConfigError("{0}: must be string or dict".format(name))
 
 
+def _process_environment(node, name):
+    # Environment can be either a list of strings ("KEY=VALUE") or a mapping
+    # Environment keys and values are always strings
+    result = {}
+
+    if not node:
+        pass
+    elif isinstance(node, dict):
+        for k, v in node.items():
+            if v is None:
+                v = os.getenv(k)
```

Comment:
Currently if a variable is unable to be expanded by the host environment, the value in the container is set to the string `None`, not an empty string. I believe it's because `os.getenv` returns None [if the requested key doesn't exist](https://docs.python.org/3.7/library/os.html#os.getenv) and the value of `str(None) = "None"`.

Reply:
This behavior also exists in `utils.parse_env_vars`.

Reply:
Good catch. I must have been thinking that `os.getenv('NONEXIST')` would return an empty string if the key doesn't exist.

Luckily, the fix is easy:
```diff
-                v = os.getenv(k)
+                v = os.getenv(k, '')
```

Reply:
Fixed in ea9259b.